tests/unit/neptune/new/test_stringify_unsupported.py

```diff
@@ -302,6 +302,35 @@ def test_assign__dict(self, run):
 
         assert run["regular"].fetch() == run["stringified"].fetch()
 
+    def test_assign__drops_dict_entry_with_practically_empty_key(self, caplog, run):
+        with assert_unsupported_warning():
+            run["unsupported"] = {"": Obj(), "x": Obj()}
+        with assert_unsupported_warning():
+            run["unsupported"] = {"///": Obj(), "x": Obj()}
+        logs = caplog.get_records("call")
```

Comment:
This looks like a pretty good pattern (catching warning) -> Could we extract it to context-manager? Check `assert_no_warnings` as a reference.

Reply:
Done